docs/Getting-Started-with-Balance-Ball.md

```diff
@@ -130,7 +130,7 @@ each element of the vector means is defined by the agent logic (the PPO
 training process just learns what values are better given particular state 
 observations based on the rewards received when it tries different values). 
 For example, an element might represent a force or torque applied to a 
-Rigidbody in the agent. The **Discrete** action vector space defines its
+Rigid body in the agent. The **Discrete** action vector space defines its
```

Comment:
I thought this was referencing the physics Rigidbody class. So maybe either "rigid body" or Rigidbody and reference https://docs.unity3d.com/Manual/class-Rigidbody.html

Reply:
Yeah. The Unity rigidbody is always treated as a single world.

Reply:
Good catch. I thought we didn't mean the word in a CS sense.  Convention should be `ClassName` if we are referencing a data structure.